master/job.go

```diff
@@ -35,20 +33,27 @@ func (j *job) decreaseRunningTasks() {
 	j.runningTasks = j.runningTasks - 1
 }
 
-// SplitJob attempts to split a cracking job into equal sized tasks regardless of the job
+// splitJob attempts to split a cracking job into equal sized tasks regardless of the job
 // the taskSize represents the number of brute force iterations
-func SplitJob(job *job, taskSize int64) []lib.Task {
-	var tasks []lib.Task
-	cands, lens := chunkCandidates(job.Alphabet, job.KeyLen, taskSize)
+func (j *job) splitJob(taskSize int64) {
+	if taskSize < int64(j.Iter) {
```

Comment:
`taskSize` is the number of SHA2 hashes. Suppose a user submits a job using Iter = 10, and another using Iter = 100, while the other parameters are the same. Then these tasks need to be split differently. The second job will have 10 times more tasks than the first job, but each of these tasks have the same size.